rsts/concepts/flyte_console.rst

```diff
@@ -0,0 +1,232 @@
+.. _ui:
+
+How to Use Flyte UI
+===================
+
+Flyte UI is a web-based user interface for Flyte. It helps interact with Flyte objects and builds DAGs out of your workflows.
+
+With Flyte UI, you can:
+
+* Launch Execution
+* Launch Task
+* View Versioned Executions
+* Trigger Versioned Executions
+* Inspect Execution through Inputs, Outputs, Logs, and Graphs
+* Clone Execution
+* Relaunch Execution
+* Recover Execution
+
+.. note::
+    `Flyte Console <https://github.com/flyteorg/flyteconsole>`__ hosts the Flyte user interface code.
+
+Launch Execution
+----------------
+
+Launch an execution by clicking on the **Launch Workflow** button. Workflows are viewable after they are registered.
+UI should be accessible at http://localhost:8000/console.
+
+|
+
+.. figure:: https://raw.githubusercontent.com/flyteorg/flyte/static-resources/img/flyteconsole/launch_execution_001.png
+    :alt: "Launch Workflow" button
+
+    Launch a workflow using the "Launch Workflow" button.
+
+|
+
+The end-to-end process from writing code to registering workflows is present in the :std:ref:`gettingstarted_implement`.
+
+After clicking on the **Launch Workflow** button, a pop-up window gets displayed with input fields that the execution requires.
+If default inputs are given, they get auto-populated.
+
+|
+
+.. figure:: https://raw.githubusercontent.com/flyteorg/flyte/static-resources/img/flyteconsole/launch_execution_002.png
+    :alt: Launch form
+
+    A pop-window gets displayed after clicking on the "Launch Workflow" button.
+
+|
+
+An execution can be terminated/aborted by clicking on the **Terminate** button.
+
+|
+
+.. figure:: https://raw.githubusercontent.com/flyteorg/flyte/static-resources/img/flyteconsole/launch_execution_003.png
+    :alt: "Terminate" button
+
+    Terminate an execution by clicking on the "Terminate" button.
+
+|
+
+Launch Task
+-----------
+
+Launch a task by clicking on the **Launch Task** button. Tasks are viewable after they are registered.
+UI should be accessible at http://localhost:8000/console.
+
+|
+
+.. figure:: https://raw.githubusercontent.com/flyteorg/flyte/static-resources/img/flyteconsole/launch_task_001.png
+    :alt: "Launch Task" button
+
+    Launch a task using the "Launch Task" button.
+
+|
+
+After clicking on the **Launch Task** button, a pop-up window gets displayed with input fields that the task requires
+and the role with which the task has to run.
+
+|
+
+.. figure:: https://raw.githubusercontent.com/flyteorg/flyte/static-resources/img/flyteconsole/launch_task_002.png
+    :alt: Launch form
+
+    A pop-window gets displayed after clicking on the "Launch Task" button.
+
+|
+
+View Versioned Executions
```

Comment:
I think this term is a bit confusing... it implies flyte versions the executions, when in fact it doesnt...

Reply:
Yes, I have replaced executions with tasks and workflows.